inst/templates/derivationchain/body.mustache

```diff
@@ -29,4 +31,5 @@ var statsImage = '{{reportData.derivationChainSymbols.statDerived}}';
 var fillMissingDataImage = '{{reportData.derivationChainSymbols.fillMissingData}}';
 var conditionalDataImage = '{{reportData.derivationChainSymbols.conditionalFill}}';
 var verticalDatumConversionImage = '{{reportData.derivationChainSymbols.verticalDatumConversion}}';
+var noProcessorImage = '{{reportData.derivationChainSymbols.noProcessor}}';
```

Comment:
@mwernimont it looks like we missed this one? or maybe it was intentional/no examples to test with? anyway, added it while I was in here, let me know if it's unknown yet and i should remove this and other changes related if so. thanks :)